lib/RTCSession.js

```diff
@@ -1009,13 +1009,28 @@ module.exports = class RTCSession extends EventEmitter
       }
       else
       {
-        const dtmf = new RTCSession_DTMF(this);
+        // send DTMF according to transport config
+        switch (this._ua.config.session_dtmf_transport.toUpperCase())
+        {
+          case JsSIP_C.DTMF_TRANSPORT.RFC2833:
+            // TODO: support payload 101, et al.
+            break;
+
+          case JsSIP_C.DTMF_TRANSPORT.IN_BAND:
+            // send DTMF in current audio RTP stream
+            this._getDTMFRTPSender().insertDTMF(tone, duration, interToneGap);
```

Comment:
This will throw everytime `_getDTMFRTPSender()` doesn't return a  `RTCDTMFSender` instance.

Also it is not notifying the user about the new sent DTMF, as [we do](http://jssip.net/documentation/3.0.x/api/session/#event_newDTMF) with the INFO DTMFs.

If `sendDTMF()` is called before a previous call to this method has finished sending all the tones, the remaining ones will be discarded and not sent.